### PR TITLE
Add LabelsToTagsMap and EnsureStringArray functions to sdk plugin. closes #3

### LINKED ISSUE #3: Add LabelsToTagsMap and EnsureStringArray functions to sdk plugin

### PR DESCRIPTION
**Pull request by tyagiparth**

### CODE REVIEW

File: plugin/transform/primitives.go [hunk 1]
```diff
@@ -396,3 +397,38 @@ func UnixMsToTimestamp(_ context.Context, d *TransformData) (interface{}, error)
...
+	case []string:
+		if labels == nil {
+			return result, nil
+		}
+		for _, i := range labels {
+			result[i] = true
+		}
+		return result, nil
+	default:
+		return nil, errors.New("ERROR")
```
Comment (kaidaguerre) [inline plugin/transform/primitives.go:431]:
```
return nil, fmt.errorf("LabelsToTagsMap transform requires the input to be []string, got %s", reflect.Typeof(d.Value).Name)
```

**Comment by e-gineer** [comment]
What name are we using for this function? Maybe it should be StringArrayToMap?

**Comment by kaidaguerre** [comment]
yup that's the question. `StringArrayToMap` is good

File: plugin/transform/primitives.go [hunk 2]
```diff
@@ -396,3 +396,38 @@ func UnixMsToTimestamp(_ context.Context, d *TransformData) (interface{}, error)
...
+			str := fmt.Sprintf("%v", d.Value)
+			return []string{str}, nil
+		}
+
+	}
+	return nil, nil
+}
+
+// LabelsToTagsMap :: converts an array raw label/tag strings into a tags map supported by steampipe
+func LabelsToTagsMap(_ context.Context, d *TransformData) (interface{}, error) {
```
Comment (kaidaguerre) [inline plugin/transform/primitives.go:418]: rename to StringArrayToMap

**Review by kaidaguerre — CHANGES_REQUESTED** [review]

**Pull request merged**